### ISSUE TITLE
copy-pasting from VSCode behavior

### ISSUE THREAD

**Issue by fpvandoorn** [nomeata/loogle#22]
If I copy-paste text from VSCode into the Loogle search bar, and the search bar is already non-empty, two unexpected things happen:
* A space is inserted before and after the pasted text (if the cursor is not at the end or the beginning of the existing search query at least)
* The cursor is placed one character before the end of the pasted string 

This behavior is easily visible if you copy text in VSCode and then paste it multiple times in the Loogle search bar.

I am using Windows & Firefox.

**Comment by nomeata** [comment]
Hmm, can’t reproduce on Firefox and Linux. Does it depend on what you are copying, e.g. if it involves unicode or backslashes or so?

**Comment by fpvandoorn** [comment]
Nope, it can be any word or a single letter, highlighted or not. I expect that it has to do with the fact that some markup is copied with it.

FWIW: I also cannot reproduce this on Windows + Chrome...

E.g. if I copy `AddAction` from VSCode, and paste it a couple times in Chrome I get
```
AddActionAddActionAddActionAddActionAddActionAddActionAddActionAddAction
```
in Firefox I get
```
AddAction AddActio AddActio AddActio AddActio AddActio AddActio
AddAction
n n n n n n
```
Only when copying it here I notice that there's even newlines in there...
